wolff/solver.py

```diff
@@ -113,6 +133,14 @@ def simulation_time(self, value):
             raise ValueError("Cannot set simulation time to a negative number")
         self._simulation_time = value
 
+    @property
+    def target_times(self):
+        return self._target_times
+
+    @target_times.setter
+    def target_times(self, value):
+        self._target_times = value
+
```

Comment:
If we do change to 'simulation_times`, we can change the setter such that if we get a scalar, we force it into a list or if we get  a list, we just assign it. Must copy the list because they are mutable in Python.
```suggestion
    @property
    def target_times(self):
        return self._target_times

    @target_times.setter
    def target_times(self, value):
        if isinstance(value, list):
            # Deep copy list
            self._target_times = list(value)
        else:
            # Assume constant number
            self._target_times =[value]

```